### PR TITLE
zboss: Add libgpcb library [KRKNWK- 13598]

### PR DESCRIPTION
**Pull request by sebastiandraus**

### CODE REVIEW

**Review by tomchy — APPROVED** [review]
Please do not forget about adding a note inside the changelog :slightly_smiling_face:

**Comment by sebastiandraus** [comment]
ZBOSS changelog updated

File: zboss/CHANGELOG.rst [hunk 1]
```diff
@@ -9,6 +9,21 @@ Changelog
...
 
+Main branch
+***********
+
+All the notable changes included in the |NCS| main branch are documented in this section.
+
+Added
+=====
+
+* Precompiled libraries for experimental Zigbee Green Power Combo Basic functionality.
```
Comment (greg-fer) [inline zboss/CHANGELOG.rst:20]: If we removed the libs for ED, let's state here "for all device roles"?

Comment (sebastiandraus) [reply zboss/CHANGELOG.rst:20]: We did not remove any functionality by removing these libs, Proxy Basic functionality is not meant for End Devices anyway. Combo Basic functionality also is not meant for End Devices but we can specify Router and Coordinator roles

File: zboss/CHANGELOG.rst [hunk 2]
```diff
@@ -9,6 +9,21 @@ Changelog
...
 
+Main branch
+***********
+
+All the notable changes included in the |NCS| main branch are documented in this section.
+
+Added
+=====
+
+* Precompiled libraries for experimental Zigbee Green Power Combo Basic functionality for router and coordinator roles.
```
Comment (greg-fer) [inline zboss/CHANGELOG.rst:20]: Router and Coordinator, both names capitalized.

Comment (sebastiandraus) [reply zboss/CHANGELOG.rst]: Done!

**Comment by sebastiandraus** [comment]
Rebased at newest main

**Review by rlubos — APPROVED** [review]

**Pull request merged**